app/client/src/utils/helpers.test.ts

```diff
@@ -6,6 +6,45 @@ import {
 } from "./helpers";
 
 describe("flattenObject test", () => {
+  it("Check if recursively referenced object is returend correctly", () => {
```

Comment:
Possibly the smallest nit possible. :) 

```suggestion
  it("Check if recursively referenced object is returned correctly", () => {
```